### PR TITLE
Secondary label examples for radio and checkbox

### PR DESCRIPTION
**Pull request by Vineet119**

### CODE REVIEW

File: site/docs/components/radio-button/examples.mdx [hunk 1]
```diff
@@ -115,5 +115,13 @@ You can wrap `RadioButtonGroup` in a form field when it’s within a form contex
 
 For more information, refer to the [form field](/salt/patterns/forms) guidance.
 
+</LivePreview>
+
+<LivePreview componentName="radio-button" exampleName="WithSecondaryLabel" >
+
+## WithSecondaryLabel
+
+You can use radioButton with secondary label in a form field.
```
Comment (origami-z) [inline site/docs/components/radio-button/examples.mdx:124]:
```suggestion
You can use radio button with secondary label in a form field.
```

Comment (Vineet119) [reply site/docs/components/radio-button/examples.mdx]: done

File: packages/core/stories/radio-button/radio-button.stories.tsx [hunk 2]
```diff
@@ -15,6 +20,19 @@ export const Checked = () => {
   return <RadioButton label="Subscribe" value="subscribe" checked />;
 };
 
+export const WithSecondaryLabel: StoryFn<typeof Checkbox> = () => {
```
Comment (joshwooding) [inline packages/core/stories/radio-button/radio-button.stories.tsx:23]:
```suggestion
export const WithSecondaryLabel: StoryFn<typeof RadioButton> = () => {
```

File: site/docs/components/checkbox/examples.mdx [hunk 3]
```diff
@@ -135,5 +135,13 @@ You can wrap checkbox groups in a form field when it’s displayed within a form
 
 For more information, refer to the [form pattern](/salt/patterns/forms).
 
+</LivePreview>
+
+<LivePreview componentName="checkbox" exampleName="WithSecondaryLabel" >
+
+## WithSecondaryLabel
+
+You can use checkbox with secondary label in a form field.
```
Comment (joshwooding) [inline site/docs/components/checkbox/examples.mdx:144]: We need to update this guidance

File: site/docs/components/radio-button/examples.mdx [hunk 4]
```diff
@@ -115,5 +115,13 @@ You can wrap `RadioButtonGroup` in a form field when it’s within a form contex
 
 For more information, refer to the [form field](/salt/patterns/forms) guidance.
 
+</LivePreview>
+
+<LivePreview componentName="radio-button" exampleName="WithSecondaryLabel" >
+
+## WithSecondaryLabel
+
+You can use radio button with secondary label in a form field.
```
Comment (joshwooding) [inline site/docs/components/radio-button/examples.mdx:124]: And here

File: packages/core/stories/checkbox/checkbox.stories.tsx [hunk 5]
```diff
@@ -46,6 +52,19 @@ export const Indeterminate: StoryFn<typeof Checkbox> = () => {
   );
 };
```
Comment (joshwooding) [inline packages/core/stories/checkbox/checkbox.stories.tsx:54]: We should have group examples too

File: packages/core/stories/radio-button/radio-button.stories.tsx (hunk 2 above)
Comment (Vineet119) [reply packages/core/stories/radio-button/radio-button.stories.tsx]: done

File: site/docs/components/checkbox/examples.mdx (hunk 3 above)
Comment (Vineet119) [reply site/docs/components/checkbox/examples.mdx]: done

File: site/docs/components/radio-button/examples.mdx (hunk 4 above)
Comment (Vineet119) [reply site/docs/components/radio-button/examples.mdx]: done

File: packages/core/stories/checkbox/checkbox.stories.tsx (hunk 5 above)
Comment (Vineet119) [reply packages/core/stories/checkbox/checkbox.stories.tsx:54]: added

**Review by origami-z — APPROVED** [review]

**Comment by joshwooding** [comment]
Before merging this I'd like to look at the site docs

**Comment by shey-v** [comment]
the row-gap should be spacing-50 (2,4,6,8) between labels and descriptions - could we update the examples please?

**Review by origami-z — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**